controllers/tasks.js

```diff
@@ -505,6 +505,17 @@ const getUsersHandler = async (req, res) => {
       return res.boom.badRequest("Unknown type and query");
     }
   } catch (error) {
+    const taskRequestLog = {
+      type: "tasksMissedUpdatesErrors",
+      meta: {
```

Comment:
Let's move this type to a constant
What is the casing type we are following at other places? is it camelCase or pascle_case or all caps, lets keep the casing of the type same overall